cmd/security-agent/common/metrics.go

```diff
@@ -0,0 +1,27 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package common
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/process/statsd"
+	"github.com/DataDog/datadog-agent/pkg/version"
+)
+
+// SendRunningMetrics exports a metric to distinguish between security-agent modules that are activated
+func SendRunningMetrics(moduleName string) {
+	// Retrieve the agent version using a dedicated package
+	tags := []string{fmt.Sprintf("version:%s", version.AgentVersion)}
+
+	// Send the metric regularly
+	heartbeat := time.NewTicker(15 * time.Second)
```

Comment:
I know that system-probe, the agent and trace-agent sends a metric every 15 seconds but it's probably a bit overkill. In the JIRA we said 5 minutes but that may be to long. Would 1 minute be a good tradeoff ?

Reply:
You are correct. I used a value similar as other parts of the agent.

I think that 15 seconds is the good value for datadog.security_agent.compliance.running, as it is the only metric that could be used to check if the compliance part is running. As such, it should the same value as other agents, to make things consistent.

datadog.security_agent.runtime.running may use a different longer time, but I find it confusing.